PreProcessing/preprocess.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+import argparse
+
+def preProcess():
+    df  =  pd.read_json('data/uniqueClassData.json')
+    df_y  =  pd.read_json('data/yearEnrollmentData.json')
+
+    df  =  df.assign(**{ '# Offerings':1, '# Y1': 0, '# Y2': 0,'# Y3': 0,'# Y4': 0,'# Y5+': 0})
+
+    for i in df.index:
+        
```

Comment:
Nit: there's also quite a fair bit of whitespace in this file at the end of lines or on blank lines. There should be a setting in the editor that you use to automatically remove trailing whitespace on saves.